packages/salesforcedx-vscode-core/test/vscode-integration/telemetry/MockExtensionContext.ts

```diff
@@ -80,7 +80,7 @@ class MockEnvironmentVariableCollection
   }
 }
 
-export class MockContext implements ExtensionContext {
+export class MockExtensionContext implements ExtensionContext {
```

Comment:
I actually do really like this change here - much clearer what/why we're mocking 👍